### DISCUSSION TITLE
Is this repo maintained?

### DISCUSSION THREAD

**Discussion by ddenev** [codesandbox/codesandbox-client#6739] [Q&A]
People are reporting problems with creating sandboxes (see the Issues) for more than a month now and there is nobody to reply and comment. Is this repo dead?

https://github.com/codesandbox/codesandbox-client/issues/6721
https://github.com/codesandbox/codesandbox-client/issues/6737
https://github.com/codesandbox/codesandbox-client/issues/6738

**Comment by JamesACS** [comment]
Hey @ddenev - This is maintained, we have just been unable to reproduce the above issues. 

Are you encountering similar?

**Reply by ddenev** [reply]
Hi @JamesACS 

Yes, I was having similar issues but they were resolved as of yesterday for which I thank you immensely! :)

**Reply by JamesACS** [reply]
No problem at all, glad to hear things are working again!
